tests/letstest/scripts/test_letsencrypt_auto_venv_only.sh

```diff
@@ -5,3 +5,8 @@
 cd letsencrypt
 # help installs virtualenv and does nothing else
 ./letsencrypt-auto-source/letsencrypt-auto -v --debug --help all
+
+if ! ./letsencrypt-auto-source/letsencrypt-auto --help --no-self-upgrade | grep -F "letsencrypt-auto [SUBCOMMAND]"; then
```

Comment:
Why are we adding this, especially in this PR?

Reply:
See above.